stage-0-proposals.md

```diff
@@ -53,3 +59,13 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [async-context]: https://github.com/legendecas/proposal-async-context
 [async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-23.md#async-context-updates--for-stage-1
 [string-trim-characters]: https://github.com/Kingwl/proposal-string-trim-characters
+[array-async-from]: https://github.com/js-choi/proposal-array-async-from
+[explicit-this-param]: https://github.com/gilbert/es-explicit-this
+[explicit-this-param-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#syntax-for-explicitly-this-argument-for-stage-1
+[this-arg-reflection]: https://github.com/hax/proposal-function-this
+[this-arg-reflection-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#function-thisargumentexpected-property
+[index-from-end]: https://github.com/hax/proposal-index-from-end
+[index-from-end-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-28.md#index-from-end-syntax
```

Comment:
this link's conclusion says "not advancing on its own", which sounds to me like it's a rejected proposal. @hax?

Reply:
I don't think it was rejected, but need to coordination with slice notation proposal, so if slice notation proposal was updated (I haven't finished pull request for it) and presented and accepted in the meeting, this proposal would also advance.

Reply:
OK - the conclusion listed in the notes implies it would be *part* of the slice notation proposal, as opposed to being a separate proposal. do you think it makes sense to list it as a stage 0 proposal for now?

Reply:
I think the previous discussion ask for coordination because of highly related syntax. Of coz it's could be merged into slice notation proposal if we really want, but actually they are individual feature (just like class fields and class static features, etc.), for example both repo have issues which have no relationship to the other.